### DISCUSSION TITLE
OBS Virtual Cam doesnt appear in Snap Camera's Sources

### DISCUSSION THREAD

**Discussion by uscheibn** [ptrumpis/snap-camera-server#56] [Q&A] [answered]
Hi,

Snap Camera Server is working very well on my PC and I'm really happy that I can continue to use it.

In the normal Snap Camera Versions before I was able to use OBS Virtual Camera as camera source but this doesn't appear anymore under "Settings" > "Choose your camera"
I have several other camera sources available, Droidcam, an external USB Webcam and also the Notebook Internal Cam can be used without Problems. 

I cleared already the cache .

OBS Virtual Camera is working and can be chosen and used by other applications e.g. the browser. 

Is there any way to get also OBS Virtual Camera to work on Snap Camera Server?

Thx

**Comment by ptrumpis** [comment, marked as answer]
It seems to work only on Mac OS not on Windows
- https://www.reddit.com/r/obs/comments/lzm0h5/virtual_camera_with_snap_camera/
- https://support.lensstudio.snapchat.com/hc/en-us/community/posts/360042411172-OBS-Virtual-Cam-not-selectable
